dll/win32/profmap/profmap.spec

```diff
@@ -0,0 +1,5 @@
+@ stdcall -stub InitializeProfileMappingApi()
```

Comment:
Might as well export `DllMain`, since Windows's version of the dll does.

Reply:
Forgot that. Thanks.